templates/fragments/001-osb-default.conf

```diff
@@ -17,8 +17,11 @@ location ~* /(s3fs-css|s3fs-js|${DRUPAL_PUBLIC_FILES_PATH})/(.*) {
     # Remove Authorization header if present.
     proxy_set_header Authorization "";
 
-    #hidegoogleheaders
+    # Intercept and suppress errors coming from bucket.
+    proxy_intercept_errors on;
+    error_page 500 404 403 /index.php?q=404;
```

Comment:
I am not sure here in which path we should redirect the user, the idea here is to deliver the Drupal 404 page.